manifests/plugin/remote_execution.pp

```diff
@@ -1,9 +1,21 @@
 # Installs foreman_remote_execution plugin
-class foreman::plugin::remote_execution {
+#
+# === Parameters:
+#
+# $dynflow_in_core:: Whether Foreman ships the Dynflow executor service (true on 1.15+)
+#                    type:Boolean
```

Comment:
This is the old style. When you want to enforce the type you can now use Puppet 4 types in the class definition: ```Boolean $dynflow_in_core = $::foreman::dynflow_in_core```. Then Puppet will actually check the type instead of relying on Kafo to do it instead.